core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -60,7 +60,7 @@ private[spark] class CoarseGrainedExecutorBackend(
 
   import CoarseGrainedExecutorBackend._
 
-  private implicit val formats = DefaultFormats
+  private implicit val formats: DefaultFormats.type = DefaultFormats
```

Comment:
Does `CoarseGrainedExecutorBackend` still need this implicit variable? Can we directly delete this line of code?

Reply:
yes,removed